TundraTD/Assets/Scripts/City/CityGates.cs

```diff
@@ -11,24 +13,34 @@ namespace City
     [RequireComponent(typeof(CityGatesUI))]
     public class CityGates : MonoBehaviour
     {
-        [SerializeField] private float cityGatesHealthPoints;
+        [SerializeField] private float maxCityGatesHealthPoints;
+        [SerializeField] private LevelJudge levelJudge;
+        private float _currentCurrentCityGatesHealthPoints;
+        private float _cityGatesHealthPercent;
         private CityGatesUI _cityGatesUI;
 
-        public float CityGatesHealthPoints
+        private float CurrentCityGatesHealthPoints
```

Comment:
Nit: Why this property is private, is it needed in another classes?

Reply:
It's not yet needed, we will make it public if necessary